StyleCop.Analyzers/StyleCop.Analyzers.Test/ReadabilityRules/SA1132UnitTests.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) Tunnel Vision Laboratories, LLC. All Rights Reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+
+namespace StyleCop.Analyzers.Test.ReadabilityRules
+{
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Analyzers.ReadabilityRules;
+    using Microsoft.CodeAnalysis.CodeFixes;
+    using Microsoft.CodeAnalysis.Diagnostics;
+    using TestHelper;
+    using Xunit;
+
+    public class SA1132UnitTests : CodeFixVerifier
+    {
+        [Theory]
+        [InlineData("private int a;")]
+        [InlineData("public event System.Action a;")]
+        public async Task TestValidDeclarationAsync(string declaration)
+        {
+            var testCode = GetTemplate(declaration);
+
+            await this.VerifyCSharpDiagnosticAsync(testCode, EmptyDiagnosticResults, CancellationToken.None).ConfigureAwait(false);
+        }
+
+        [Theory]
+        [InlineData("private int a, b;", "private int a;\r\n    private int b;")]
```

Comment:
:thought_balloon: I find this hard to read. I think I would have preferred two separate test methods instead.